lib/model/yake_keywords.py

```diff
@@ -26,15 +54,25 @@ def run_yake(self, text: str,
         :param num_of_keywords: int
         :returns: str
         """
+        ### if language is set to "auto", auto-detect it.
+        if language == 'auto':
+            language = cld3.get_language(text).language
```

Comment:
Do we need a `try...except` around this? Have you tried with unsupported languages / nonsense and ensured this doesn't blow up?

Reply:
I don't think we need try....except. I've tried with unsupported language Ewe and nonsense text. They both got a wrong language detected by cld3 but it didn't break. 
For Yake, if you give it a wrong language or nonsupported one or you write something nonsense like "sdadsd" as the language code, the performance decreases but it doesn't break.